### ISSUE TITLE
Стиль написания кода

### ISSUE THREAD

**Issue by dsokolov** [AltarixExercises2016/Semenov.Matvey#4]
Имена пакетов следует писать с маленькой буквы https://google.github.io/styleguide/javaguide.html#s5.2.1-package-names

Здесь же:
пустые строки перед методами
в классах сначала поля, потом методы
имена классов с большой буквы( класс link)
имена полей в кемелКейсе (а не через_подчёркивание, как напирмер в DialogMessageActivity)
имена переменных так же в кемелКейсе (напирмер Token в StartActivity)

Вообще используй автоформатирование ctrl+alt+L

**Comment by matek3022** [comment]
да, спасибо, я поправлю, просто не акцентировал на этом пока что внимание)

**Comment by matek3022** [comment]
Готово :)
